nautobot/core/api/metadata.py

```diff
@@ -144,16 +192,25 @@ def determine_actions(self, request, view):
                 view.request = request
         return actions
 
-    def determine_view_options(self, request, view, serializer):
+    def get_list_display_fields(self, serializer):
+        """Try to get the list display fields or default to an empty list."""
+        serializer_meta = getattr(serializer, "Meta", None)
+        return list(getattr(serializer_meta, "list_display_fields", []))
```

Comment:
Probably should default to something like `serializer.get_fields()` instead of an empty list?